scripts/build_erddap_catalog.py

```diff
@@ -137,6 +147,26 @@ def build_erddap_catalog_fragment(data_root, user, deployment, template_dir,
                        'time_qc': 'precise_time_qc',
                        'profile_time_qc': 'time_qc'}
 
+
+    existing_varnames = {'trajectory', 'wmo_id', 'profile_id', 'profile_time',
+                         'profile_lat', 'profile_lon', 'time', 'depth',
+                         'pressure', 'temperature', 'conductivity', 'salinity',
+                         'density', 'lat', 'lon', 'time_uv', 'lat_uv',
+                         'lon_uv', 'u', 'v', 'platform', 'instrument_ctd'}
+
+    # need to explicitly cast keys to set in Python 2
+    exclude_vars = (existing_varnames | set(dest_var_remaps.keys()) |
+                    required_qc_vars | {'latitude', 'longitude'})
+
+    # need to pass None to constructor?  Why?
+    standard_name_table = util.StandardNameTable(None)
```

Comment:
We need to think about how we can make sure we're always using the latest standard name table

Reply:
It should already be doing that: https://github.com/ioos/compliance-checker/blob/master/compliance_checker/cf/util.py#L263-L271

Reply:
That doesn't guarantee the latest standard name table. There is however a way to download a specific version but how would we know what the latest version is? 

https://github.com/ioos/compliance-checker/blob/master/cchecker.py#L107-L108

Need to scrape the CF conventions page?
http://cfconventions.org/standard-names.html